icubam/www/token.py

```diff
@@ -22,3 +23,36 @@ def decode(self, token: str):
 
   def encode_data(self, user, icu) -> str:
     return self.encode((user.user_id, icu.icu_id))
+
+  def authenticate(self, token: str, db) -> Tuple:
```

Comment:
So this is identical to `decode_token` previously, right?

Reply:
absolutely!